0/mdav2/geom.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+import masses
+
+class Molecule:
+    """
+    defines the Molecule class. Molecule objects are initialized with a .xyz
+    filename as input. 
+    """
+    def __init__(self,filename):
+        self.moleculefile = open(filename,'r')
```

Comment:
It is recommended to use [`with` to open files](https://stackoverflow.com/questions/31334061/file-read-using-open-vs-with-open). This will take care of automatically closing them for you.